compiler/rustc_passes/src/stability.rs

```diff
@@ -938,7 +938,7 @@ pub fn check_unused_or_stable_features(tcx: TyCtxt<'_>) {
     if !remaining_lib_features.is_empty() {
         check_features(&mut remaining_lib_features, &local_defined_features);
 
-        for &cnum in &*tcx.crates() {
+        for &cnum in tcx.crates() {
```

Comment:
Is the leading & still necessary?

Reply:
`tcx.crates()` is a slice (`&'tcx [CrateNum]`), so I guess we either have to write `&cnum` or `*tcx.crates()`.